publish/kafka/publish.go

```diff
@@ -70,6 +75,7 @@ type Partitioner interface {
 
 func init() {
 	flag.StringVar(&topicsStr, "metrics-topic", "mdm", "topic for metrics (may be given multiple times as a comma-separated list)")
+	flag.StringVar(&rewriteOrgIdStr, "rewrite-org-id", "", "rewrite org id; for example 33:45 means rewriting org id 33 into 45 (may be given multiple times, once per topic, as a comma-separated list)")
```

Comment:
do i understand it correctly that the number of rewrites must be the same as the number of topics? if so i think the help message could be a bit more explicit about it. 

```suggestion
	flag.StringVar(&rewriteOrgIdStr, "rewrite-org-id", "", "rewrite org id; for example 33:45 means rewriting org id 33 into 45 (may be given multiple times, if given then there must be exactly one per topic, as a comma-separated list)")
```